doc/classes/AudioBusLayout.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioBusLayout" inherits="Resource" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Stores the information about the audiobusses.
 	</brief_description>
 	<description>
+		Stores position, muting, solo, bypass, effects, effectposition, volume and the way the busses input into each other. [codeblock]See [AudioServer] for usage.[/codeblock]
```

Comment:
Should be "effect position" and you need a comma after volume.  

Also, "the way the busses input into each other" sounds awkward. How about "and the connections between busses."